components/ui-shell/index.ts

```diff
@@ -47,12 +47,13 @@ export class QiskitUIShell extends LitElement {
             }}"
           >
             ${qiskitLogoIcon}
+            <div class="qiskit-header-brand">Community</div>
           </bx-header-name>
           <bx-header-nav menu-bar-label="Qiskit">
             ${this._getHeaderItems()}
             ${this.variant === Variant.HIDE_ACCOUNT
-              ? null
-              : this._getAccountHeaderNavItem()}
+              ? this._getAccountHeaderNavItem()
+              : null}
```

Comment:
Since we won't have the `hide account` option, you can delete these lines and the variant on lines 30-31